data.sql

```diff
@@ -12,3 +12,24 @@ VALUES ('Pikachu', 3,'2021-1-7',1,false,15.04);
 -- Animal:Devimon
 INSERT INTO animals (name, id, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', 4,'2017-5-12',5,true,11.0);
+-- Animal:Charmander
+INSERT INTO animals (name, id, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', 5,'2020-2-8',0,false,11.0);
+-- Animal:Plantmon
+INSERT INTO animals (name, id, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', 6,'2021-11-15',2,true,5.7);
+-- Animal:Squirtle
+INSERT INTO animals (name, id, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', 7,'1993-4-2',3,false,12.13);
+-- Animal:Angemon
+INSERT INTO animals (name, id, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', 8,'2005-6-12',1,true,45.0);
+-- Animal:Boarmon
```

Comment:
- [ ] Hi @MasumaJaffery :wave: You seem really good with Maths and that's impressive! I am certain that you will opportunities where that will be greatly useful but you see, for now, according to my colleague @joskalenda, the project requirement was clear to use the values you were given without any expected counter-multiplication or reverse-engineering. Kindly check the requirement attached below and find that there was no additional information on multiplying with a negative value to convert it to positive values:

![image](https://github.com/MasumaJaffery/Vet-Clinic/assets/99715640/9b921c7b-7724-41a6-a2e3-737623d7809d)

Kindly use the figures as seen :pray: